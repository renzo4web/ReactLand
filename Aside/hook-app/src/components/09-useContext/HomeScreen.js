import React from "react";
import ButtonToggle from "./ButtonToggle";

const HomeScreen = () => {
  return (
    <div>
      <h1>Home</h1>
      <p>Welcome</p>
      <ButtonToggle />
    </div>
  );
};

export default HomeScreen;
