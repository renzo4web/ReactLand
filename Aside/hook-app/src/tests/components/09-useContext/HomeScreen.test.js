import { shallow } from "enzyme";
import HomeScreen from "../../../components/09-useContext/HomeScreen";

describe("<HomeScreen /> Test", () => {
  const wrapper = shallow(<HomeScreen />);

  test("should ", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
