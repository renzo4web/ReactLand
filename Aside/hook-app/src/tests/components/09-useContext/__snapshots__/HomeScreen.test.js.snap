// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> Test should  1`] = `
<div>
  <h1>
    Home
  </h1>
  <p>
    Welcome
  </p>
  <ButtonToggle />
</div>
`;
